stable/kibana/templates/deployment.yaml

```diff
@@ -31,8 +31,6 @@ spec:
         imagePullPolicy: {{ .Values.image.pullPolicy }}
         {{- if .Values.commandline.args }}
         args:
-          - "/bin/bash"
-          - "/usr/local/bin/kibana-docker"
```

Comment:
Just straight up removing these could have some pretty adverse consequences. I suspect you're submitting this change because you have a custom docker image which changes the command / entrypoint so this is getting in your way? If so can we wrap the removal with something like `{{ if (not (eq (len args) 0))) }}`? 

Also I'd like to see a noted in the README around the table which notes that adding commandline arguments will require the addition of something similar to ^^^ .